images/docker/cbdb/build/ubuntu22.04/Dockerfile

```diff
@@ -0,0 +1,203 @@
+# Dockerfile for Apache Cloudberry Base Environment
+# --------------------------------------------------------------------
+# This Dockerfile sets up a Ubuntu jammy 22.04 -based container to serve as
+# a base environment for evaluating the Apache Cloudberry. It installs
+# necessary system utilities, configures the environment for SSH access,
+# and sets up a 'gpadmin' user with sudo privileges. The Apache Cloudberry
+# DEB can be installed into this container for testing and
+# functional verification.
+#
+# Key Features:
+# - Locale setup for en_US.UTF-8
+# - SSH daemon setup for remote access
+# - Essential system utilities installation
+# - Separate user creation and configuration steps
+#
+# Security Considerations:
+# - This Dockerfile prioritizes ease of use for functional testing and
+#   evaluation. It includes configurations such as passwordless sudo access
+#   for the 'gpadmin' user and SSH access with password authentication.
+# - These configurations are suitable for testing and development but
+#   should NOT be used in a production environment due to potential security
+#   risks.
+#
+# Usage:
+#   docker build -t cloudberry-db-base-env .
+#   docker run -h cdw -it cloudberry-db-base-env
+# --------------------------------------------------------------------
+
+FROM ubuntu:jammy
+
+ARG accessKeyId
+ARG secretAccessKey
+ARG bucketName
+
+# Argument for configuring the timezone
+ARG TIMEZONE_VAR="Europe/London"
+
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN stat -fc %T /sys/fs/cgroup/
+
+RUN ln -snf /usr/share/zoneinfo/Europe/London /etc/localtime && echo $TIMEZONE_VAR > /etc/timezone
+
+# --------------------------------------------------------------------
+# Install Development Tools and Utilities
+# --------------------------------------------------------------------
+RUN apt-get update -o Acquire::AllowInsecureRepositories=true && apt-get install -y --no-install-recommends --allow-unauthenticated \
+  build-essential \
+  libssl-dev \
+  gnupg \
+  openssl \
+  debhelper \
+  libfakeroot \
+  debootstrap \
+  devscripts \
+  make \
+  equivs \
+  bison \
+  ca-certificates-java \
+  ca-certificates \
+  cmake \
+  curl \
+  cgroup-tools \
+  flex \
+  gcc-11 \
+  g++-11 \
+  g++-11-multilib \
+  git \
+  krb5-multidev \
+  libapr1-dev \
+  libbz2-dev \
+  libcurl4-gnutls-dev \
+  libpstreams-dev \
+  libxerces-c-dev \
+  libsystemd-dev \
+  libevent-dev \
+  libkrb5-dev \
+  libldap2-dev \
+  libperl-dev \
+  libreadline-dev \
+  libssl-dev \
+  libxml2-dev \
+  libyaml-dev \
+  libzstd-dev \
+  libaprutil1-dev \
+  libpam0g-dev \
+  libpam0g \
+  libcgroup1 \
+  libyaml-0-2 \
+  libldap-2.5-0 \
+  libssl3 \
+  ninja-build \
+  python-setuptools \
+  python3-setuptools \
+  quilt \
+  unzip \
+  wget \
+  zlib1g-dev \
+  libuv1-dev \
+  libgpgme-dev \
+  libgpgme11 \
+  python2.7 \
+  python2.7-dev \
+  pkg-config \
+  python3.10 \
+  python3.10-dev \
+  python3-distutils \
+  sudo
```

Comment:
after testing, some packages are still needed to install for making ./configure and demo culseter successful:

```
liblz4-dev
iputils-ping 
iproute2 
rsync 
less
openssh-client
openssh-server
```